examples/spectrahedra/README.md

```diff
@@ -0,0 +1,190 @@
+# Examples for Spectrahedra
```

Comment:
Please pull changes from develop branch. The Readme file is much more short now.